src/js/App/Feedback/Feedback.scss

```diff
@@ -11,3 +13,87 @@ button.pf-c-button.chr-c-button-feedback {
     background: var(--pf-global--palette--purple-200);
   }
 }
+
+.pf-c-content {
+  margin-bottom: var(--pf-c-content--MarginBottom);
+}
+
+.chr-c-feedback-modal {
+  margin: 0;
+  padding: 0;
+  & .pf-c-modal-box__body {
+    &:last-child {
+      padding-bottom: 0;
+    }
+  }
+}
+
+.chr-c-feedback-cards {
+  margin: 8px;
+}
+
+.chr-c-feedback-card-title {
+  color: var(--pf-global--link--Color);
+}
+
+.chr-c-feedback {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
+.chr-c-feedback-content {
+  width: 75%;
+  display: flex;
+  flex-direction: column;
+  padding-right: 24px;
+  margin: 16px 16px 8px 16px;
+  padding: 8px;
+}
+
+.chr-c-feedback-button {
+  margin: 8px;
+  text-align: left;
+  margin-top: 48px;
+}
+
+.chr-c-feedback-buttons {
+  margin: 8px;
+  margin-top: 16px;
+}
+
+.chr-c-feedback-success-content {
+  margin-top: 125px;
+  margin-bottom: 125px;
+  display: flex;
+  width: 75%;
+  justify-items: center;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.pf-c-modal-box__body {
+  padding: 0;
+  margin: 0;
+}
+.pf-c-modal-box > .pf-c-button + *{
+  margin-right: 0;
+}
+.chr-c-feedback-image {
+  width: 30%;
+  height: 100%;
+}
+
+.chr-c-feedback-email {
+  margin-top: 16px;
```

Comment:
Spacing again. You can use the pf-u- classnames directly on the elements. https://www.patternfly.org/v4/utilities/spacing